src/chat_statistic/statistic.py

```diff
@@ -0,0 +1,75 @@
+import json
+from pathlib import Path
+from typing import Union
+
+import arabic_reshaper
+# from bidi.algorithm import get_display
+from hazm import Normalizer, word_tokenize
+from loguru import logger
+from src.data import DATA_DIR
+from wordcloud import WordCloud
+
+
+class ChatStatistic():
+    """Generate Chat Statistic From Telegram Chat Json File
+    """
+    def __init__(self, chat_json: Union[str, Path]):
+        """
+        Args:
+            chat_json: Path To Telegram Export Json File
+        """
+        # load json data
+        logger.info(f"loading chat data from {chat_json}")
+        with open(chat_json) as f:
+            self.chat_data = json.load(f)
+        self.normalizer = Normalizer()
+
+        # load stopwords
+        logger.info(f"loadinf stop words from {DATA_DIR/'stopwords.txt'}")
+        stopwords = open(DATA_DIR / "stopwords.txt").readlines()
+        stopwords = list(map(str.strip, stopwords))
+        self.stopwords = list(map(self.normalizer.normalize, stopwords))
+
+    def generate_word_cloud(
+        self,
+        output_dir: Union[str, Path],
+        width: int = 800, height: int = 600,
+        max_font_size: int = 250,
+    ):
```

Comment:
Better to add optional wordcloud arguments such as `background_color`.
```suggestion
    def generate_word_cloud(
        self,
        output_dir: Union[str, Path],
        width: int = 800, height: int = 600,
        max_font_size: int = 250,
        background_color = "white",
    ):
```